pom.xml

```diff
@@ -67,11 +38,11 @@
     </license>
   </licenses>
 
-    <pluginRepositories>
-        <pluginRepository>
-            <id>repo.jenkins-ci.org</id>
-            <url>http://repo.jenkins-ci.org/public/</url>
-        </pluginRepository>
-    </pluginRepositories>
+  <pluginRepositories>
+    <pluginRepository>
+      <id>repo.jenkins-ci.org</id>
+      <url>http://repo.jenkins-ci.org/public/</url>
```

Comment:
Recommended to use the same PR to change the `repositories` and `pluginRepositories` to https (see [INFRA-588](https://issues.jenkins-ci.org/browse/INFRA-588))